### ISSUE TITLE
pyjwt iat bug

### ISSUE THREAD

**Issue by dsschult** [WIPACrepo/rest-tools#74]
Remember to unpin when pyjwt fixes their bug.

**Comment by ric-evans** [comment]
Tried today, still failing with `2.6.0`

**Comment by dsschult** [comment]
Yeah, this isn't in the 2.6.0 release.  Still sitting on their master branch waiting for a release.
